app/models/quiz_participation.rb

```diff
@@ -0,0 +1,7 @@
+class QuizParticipation < ApplicationRecord
+  belongs_to :student, class_name: "User"
+  belongs_to :quiz
+
+  validates :marks_obtained, presence: true, numericality: { only_integer: true }
+  validates :total_marks, presence: true, numericality: { only_integer: true }
```

Comment:
why only integer?

Reply:
Apu, currently considering only integer marks for each questions.